exampleSite/staticman.yml

```diff
@@ -0,0 +1,21 @@
+comments:
+  allowedFields: ['author', 'content', 'email', 'parent_id', 'permalink', 'site']
+  branch: 'master'
+  commitMessage: "add [comment]: by {fields.author} <Staticman>\n\n{fields.permalink}#comment-{@id}"
+  filename: '{@id}'
+  format: 'yaml'
+  generatedFields:
+    date:
+      type: date
+      options:
+        format: 'timestamp'
+  moderation: false
+  name: 'Minimo'
+  path: 'data/comments/{options.postId}'
+  requiredFields: ['author', 'content', 'email', 'permalink']
+  reCaptcha:
```

Comment:
Sorry, won't do. I want people to visit Staticman's documentation.

Reply:
Staticman's documentations is **outdated** because it only covers v2.  Compare the info from https://staticman.net with what Staticman's author says in https://github.com/eduardoboucas/staticman/issues/243#issuecomment-453754860.  @staticmanlab and @BloggerBust-bot don't implement this due to eduardoboucas/staticman#283 though.